test/helper/e2e/actions/steps.go

```diff
@@ -130,6 +131,10 @@ func WaitTestApplication(data *model.TestDataProvider, ns, labelKey, labelValue
 	// temp
 	isAppRunning := func() func() bool {
 		return func() bool {
+			status, err := k8s.GetPodStatus(data.Context, data.K8SClient, ns)
```

Comment:
Are these debugging bits? if not, could you clarify how it supports the second status check (line 138)?

Reply:
yes, I am debugging, please ignore these for the time being.

I will mark as draft and iterate in the `helm-update` error first, then will try to move to check the issues within ginkgo instead of from a script, as you suggested above.

Reply:
Removed the traces, thinking about a different way to debug